src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -188,435 +188,730 @@ If a problem occurs, check output in /var/log/proactive/scheduler
-    environment:
-      - PROTOCOL
-      - PORT
-      - DB_TYPE=mysql
-    volumes:
-      - proactive-default:/opt/proactive/server/default
-      - proactive-previous:/opt/proactive/server/previous
-    depends_on:
-      proactive-database:
-        condition: service_healthy
-    healthcheck:
-        test: ["CMD", "curl", "-k", "-f", "${PROTOCOL}://proactive-server:${PORT}"]
-        interval: 30s
-        timeout: 10s
-        retries: 30
-
-  proactive-database:
-    restart: on-failure
-    image: dockerhub.activeeon.com/${RELEASE}/mysql-pa:8.0
-    container_name: proactive-database
-    networks:
-      - pa-network
-    env_file:
-      - .env
-    environment:
-      - MYSQL_ROOT_PASSWORD
-    volumes:
-      - proactive-data:/var/lib/mysql
-    ports:
-      - 3306:3306
-    healthcheck:
-        test: [ "CMD", "mysqladmin" ,"ping", "-h", "proactive-database", "-u", "root", "-p${MYSQL_ROOT_PASSWORD}" ]
-        interval: 20s
-        timeout: 10s
-        retries: 10
-
-  proactive-node:
-    image: dockerhub.activeeon.com/${RELEASE}/proactive-node:${PROACTIVE_VERSION}
-    hostname: service-node
-    container_name: proactive-node
-    networks:
-      - pa-network
-    env_file:
-      - .env
-    # enable DOCKER_HOST on Windows machine when you want to run docker-in-docker
-    #environment:
-    #    - DOCKER_HOST=tcp://host.docker.internal:2375
-    volumes:
-      - /tmp
-      - /var/run/docker.sock:/var/run/docker.sock:rw
-      - node-default:/opt/proactive/node/default
-      - node-previous:/opt/proactive/node/previous
-    depends_on:
-      proactive-server:
-        condition: service_healthy
 
+You need then to reference the folders paths in the Docker Compose configuration of the `docker-compose-pg.yml` file:
+
+----
+...
 volumes:
   proactive-default:
+    ...
+      device: '/opt/proactive/default'
   proactive-previous:
-  node-default:
-  node-previous:
+    ...
+      device: '/opt/proactive/previous'
+  proactive-backup:
+    ...
+      device: '/opt/proactive/backup'
   proactive-data:
-
-networks:
-  pa-network:
+    ...
+      device: '/opt/proactive/data'
 ----
 
-==== Start ProActive using Docker Compose
+==== Start ProActive Using Docker Compose
 
-Login into the Private Activeeon Registry
+Login into the private Activeeon registry:
 ----
 $ docker login dockerhub.activeeon.com
 ----
 
-Start ProActive container with a default HSQL Database:
+To start the ProActive server with a default HSQL database, run:
 ----
 $ docker-compose -f docker-compose-hsqldb.yml up -d
 ----
 
-Start ProActive container with a PostgreSQL Database and Node containers:
+To start the ProActive server with a PostgreSQL database, run:
 ----
-$ docker-compose -f docker-compose-postgres.yml up -d
+$ docker-compose -f docker-compose-pg.yml up -d
 ----
 
-Start ProActive container with a MySQL Database and Node containers:
+Wait until the `docker logs -f proactive-scheduler` command output shows that ProActive is started:
```

Comment:
To monitor the ProActive server startup, run:
`docker logs -f proactive-scheduler` 
Wait until the command output shows that ProActive is started:

Reply:
ok